frontend/src/components/worksheets/BundleDetail/BundleDetailSideBar.jsx

```diff
@@ -146,6 +147,13 @@ class BundleDetailSideBar extends React.Component<{
                     )}
                 </div>
                 {/** ----------------------------------------------------------------------------------------------- */}
+                {isRunBundle && runStatus && (
+                    <div>
+                        <ConfigLabel label='Run status: ' inline={true} />
+                        <div className={classes.dataText}>{runStatus}</div>
```

Comment:
Will it always be red in color?

Reply:
It won't ever be red in color (same as shown in the screenshot).